pubsub/topic.go

```diff
@@ -474,6 +474,14 @@ func (t *Topic) Stop() {
 	t.scheduler.FlushAndStop()
 }
 
+// Flush ensures all remaining messages are sent.
```

Comment:
Maybe this is implied, but should we explicitly say that this method is blocking?